### ISSUE TITLE
[issue] squirrel cache incorrect display

### ISSUE THREAD

**Issue by tombabcode** [EnderWiggin/hafen-client#135]
Sometimes squirrel caches that are empty (green marked in-game, icon is not visible on the map) turns out to be full. They suddenly become red when I open the cache

**Comment by EnderWiggin** [comment]
Yes, this is because server does not initialize them before you interact with them. I've notified loftar, but not sure he will do anything about it.
